pkg/cmd/target/target.go

```diff
@@ -249,14 +246,12 @@ func (o *TargetOptions) Complete(f util.Factory, cmd *cobra.Command, args []stri
 		}
 	}
 
-	o.ControlPlane = tf.ControlPlane()
-
 	return nil
 }
 
 // Validate validates the provided options
 func (o *TargetOptions) Validate() error {
-	if o.ControlPlane {
+	if o.Kind == TargetKindControlPlane {
```

Comment:
let's discuss about the change in the validate function (in general)

Reply:
I agree. I propose:
```go
switch o.Kind {
case TargetKindControlPlane:
  break
case "":
  return error.New("error")  
default: 
  if o.TargetName == "" {
    return error.New("error")  
  } 
}
```